package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Clone of omegle.com",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "postinstall": "node node_modules/bower/bin/bower install"
```

Comment:
can you please change this?
```"postinstall": "node_modules/bower/bin/bower install"```

Reply:
Also why did you remove the test script. keep it as it was. we ll add tests later.